blockchain/src/chain_store.rs

```diff
@@ -272,6 +272,36 @@ impl ChainStore {
         }
     }
 
+    pub fn get_block_hashes_at(
+        &self,
+        block_height: u32,
+        txn_option: Option<&Transaction>,
+    ) -> Result<Vec<Blake2bHash>, BlockchainError> {
```

Comment:
As far as I can see, the function actually cannot return an error. So, a return type of `Vec<Blake2bHash>` would be sufficient.

Reply:
Yes, you are completely right. Modified in accordance